agents/ocf/HealthCPU.in

```diff
@@ -127,13 +127,27 @@ healthcpu_monitor() {
 
         if [ $IDLE -lt ${OCF_RESKEY_red_limit} ] ; then
             attrd_updater -n "#health-cpu" -B "red" -d "${OCF_RESKEY_dampening}"
+            rc=$?
+
+            if [ $rc -ne 0 ]; then
+                ocf_log err "Could not set #health-cpu attribute: $(crm_error -X $rc)"
+                return $OCF_DEGRADED
```

Comment:
Likewise, perhaps these should just be errors since this is one of the things that the agent is explicitly supposed to do; no strong feelings. I see the rationale since the idleness is under the threshold.

Similar comments apply to other agents